desktop-widgets/preferences/preferences_defaults.h

```diff
@@ -16,7 +16,6 @@ class PreferencesDefaults : public AbstractPreferencesWidget {
 	~PreferencesDefaults();
 	void refreshSettings() override;
 	void syncSettings() override;
-//	void on_resetSettings_clicked();
```

Comment:
again, if you had just removed these in the first place, your commits would be a lot less noisy and easier to follow.